include/drivers/pinctrl.h

```diff
@@ -235,7 +235,7 @@ struct pinctrl_dev_config {
  * @param node_id Node identifier.
  */
 #define PINCTRL_DT_DEFINE(node_id)					       \
-	UTIL_LISTIFY(DT_NUM_PINCTRL_STATES(node_id),			       \
+	LISTIFY(DT_NUM_PINCTRL_STATES(node_id),				       \
 		     Z_PINCTRL_STATE_PINS_DEFINE, node_id)		       \
```

Comment:
Isn't this missing the entry separator character?

Reply:
fixed.